docs/known-issues.md

```diff
@@ -9,3 +9,4 @@ Known issue = any major user-facing bug or lack of user-facing feature that:
 | IL2CPP does not currently work on Mac as it crashes upon starting the game. This is due to a bug in Unity that does not initialize a system correctly, if one of the injected `IComponentData` contains a `long`.                                                                                                            | 2018/08/31 | 1076596                                                                                                | Use the Mono compiler.                                                |
 | There is a bug in the preview version of the [High Definition Render Pipeline (Unity Blog)](https://blogs.unity3d.com/2018/03/16/the-high-definition-render-pipeline-focused-on-visual-quality/), where shaders do not fully compile and appear visually darker than intended.                                               | 2018/10/12 | [566245](https://forum.unity.com/threads/shadergraph-subgraph-importing-bug-on-shared-project.566245/) | Manually reimport shaders.                                            |
 | The Unity Editor doesn't correctly import navmeshes.                                                                                                                                                                                                                                                                         | 2018/10/12 | None                                                                                                   | Manually regenerate the navmesh after opening the project.            |
+| Error: Project file has no matching compilation unit                                                                                                                                                                                                                                                                         | 2018/11/02 | [1059037](https://issuetracker.unity3d.com/issues/project-file-has-no-matching-compilation-unit)                                                                                                   | Install [these upgrades](https://issuetracker.unity3d.com/issues/project-file-has-no-matching-compilation-unit) from Microsoft.            |
```

Comment:
this sounds like it would be better in the troubleshooting section as it means that people installed a too old VS version (might also be good to check the setup guide whether we specify the VS version)

Reply:
We have a warning in the setup guide about older versions of VS.

We did have a specific version that was verified but we removed it for some reason